packages/perseus/src/widgets/radio/choice.new.tsx

```diff
@@ -1,273 +1,50 @@
-                                            }
-                                            reviewMode={reviewMode}
-                                        />
-                                    </div>
-                                    <div>{content}</div>
-                                </span>
-                            </div>
-                        )}
-                    </Clickable>
-                </div>
-                {showRationale && (
-                    <div
-                        className={rationaleClassName}
-                        style={styles.rationale}
-                        data-testid={`perseus-radio-rationale-content-${pos}`}
-                    >
-                        {rationale}
-                    </div>
-                )}
-            </div>
-        );
-    },
-);
-
-const styles = {
-    description: {
-        display: "inline-block",
-        width: "100%",
-        flexDirection: "column",
-        color: color.offBlack,
-    },
-
-    choicesContainer: {
-        display: "flex",
-    },
-
-    optionContainer: {
-        flex: 1,
-        color: color.offBlack,
-        userSelect: "text",
-    },
-
-    choiceRow: {
-        display: "flex",
-        flexDirection: "row",
-        justifyContent: "flex-start",
-        alignItems: "center",
-        paddingBlock: spacing.xSmall_8,
-        paddingInlineStart: spacing.xSmall_8,
-        position: "relative",
-        minWidth: "100%",
-        insetInlineStart: 0,
-        // The height and background will ensure content will be scrolled
-        // behind the ChoiceIcon and will not be visible
-        height: "100%",
-        background: color.white64, // TODO: LEMS-3108 address light/dark mode theme
-    },
-
-    choiceIcon: {
-        flexGrow: 0,
-        flexShrink: 0,
-        flexBasis: "30px",
-        position: "sticky",
-        insetInlineStart: 0,
-        background: "linear-gradient(to right, white 70%, transparent)",
-        // TODO: LEMS-3107 not properly working fix this by moving in a css file
-        ":dir(rtl)": {
-            background: "linear-gradient(to left, white 70%, transparent)",
-        },
-        paddingInlineStart: spacing.xSmall_8,
-        paddingInlineEnd: spacing.xSmall_8,
-        paddingTop: spacing.xxxSmall_4,
-        zIndex: 1,
-        alignItems: "center",
-        justifyContent: "center",
-        height: "100%",
-    },
-
-    choiceContent: {
-        paddingInlineStart: spacing.small_12,
-        textAlign: "start",
-        flex: 1,
-        paddingTop: spacing.xxxSmall_4,
-        minWidth: 0,
-    },
-
-    rationale: {
-        display: "block",
-        padding: intermediateCheckboxPadding,
-        paddingTop: 0,
-        marginInlineStart: 54,
-        color: color.offBlack64,
-        [mediaQueries.smOrSmaller]: {
-            padding: intermediateCheckboxPaddingPhone,
-            paddingTop: 0,
-        },
-    },
-} as const;
+const Choice = (props: IndicatorProps) => {
```

Comment:
![step-brothers-room-for-activities](https://github.com/user-attachments/assets/12dfd317-ab1b-4bc6-a083-57dffe48ea9a)

Look at how much more svelte this component is! Great work